src/frontend/src/app/view/assements/kube-bench-report-list/kube-bench-report-list.component.spec.ts

```diff
@@ -1,19 +1,77 @@
ARN","actual_value":"","scored":false,"IsMultiple":false,"expected_result":"","reason":"Test marked as a manual test"}]},{"section":"5.7","type":"","pass":0,"fail":0,"warn":4,"info":0,"desc":"General Policies","results":[{"test_number":"5.7.1","test_desc":"Create administrative boundaries between resources using namespaces (Manual)","audit":"","AuditEnv":"","AuditConfig":"","type":"manual","remediation":"Follow the documentation and create namespaces for objects in your deployment as you need\nthem.\n","test_info":["Follow the documentation and create namespaces for objects in your deployment as you need\nthem.\n"],"status":"WARN","actual_value":"","scored":false,"IsMultiple":false,"expected_result":"","reason":"Test marked as a manual test"},{"test_number":"5.7.2","test_desc":"Ensure that the seccomp profile is set to docker/default in your pod definitions (Manual)","audit":"","AuditEnv":"","AuditConfig":"","type":"manual","remediation":"Use security context to enable the docker/default seccomp profile in your pod definitions.\nAn example is as below:\n  securityContext:\n    seccompProfile:\n      type: RuntimeDefault\n","test_info":["Use security context to enable the docker/default seccomp profile in your pod definitions.\nAn example is as below:\n  securityContext:\n    seccompProfile:\n      type: RuntimeDefault\n"],"status":"WARN","actual_value":"","scored":false,"IsMultiple":false,"expected_result":"","reason":"Test marked as a manual test"},{"test_number":"5.7.3","test_desc":"Apply Security Context to Your Pods and Containers (Manual)","audit":"","AuditEnv":"","AuditConfig":"","type":"manual","remediation":"Follow the Kubernetes documentation and apply security contexts to your pods. For a\nsuggested list of security contexts, you may refer to the CIS Security Benchmark for Docker\nContainers.\n","test_info":["Follow the Kubernetes documentation and apply security contexts to your pods. For a\nsuggested list of security contexts, you may refer to the CIS Security Benchmark for Docker\nContainers.\n"],"status":"WARN","actual_value":"","scored":false,"IsMultiple":false,"expected_result":"","reason":"Test marked as a manual test"},{"test_number":"5.7.4","test_desc":"The default namespace should not be used (Manual)","audit":"","AuditEnv":"","AuditConfig":"","type":"manual","remediation":"Ensure that namespaces are created to allow for appropriate segregation of Kubernetes\nresources and that all new resources are created in a specific namespace.\n","test_info":["Ensure that namespaces are created to allow for appropriate segregation of Kubernetes\nresources and that all new resources are created in a specific namespace.\n"],"status":"WARN","actual_value":"","scored":false,"IsMultiple":false,"expected_result":"","reason":"Test marked as a manual test"}]}],"total_pass":0,"total_fail":0,"total_warn":26,"total_info":0,"createTime":"2023-02-15T15:19:14Z","node_name":"inspectionpolicy-sample-kubebench-daemonset-s2dql"},"sort":[1676474354000]}]}})
```

Comment:
Can we further simplify this huge line?

The best practice is that one line should not exceed 100 chars.

Can we put this content into a file or further simplify it?